src/main/java/ch/njol/skript/expressions/ExprNode.java

```diff
@@ -0,0 +1,142 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.classes.ClassInfo;
+import ch.njol.skript.config.Node;
+import ch.njol.skript.config.SectionNode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.ParseContext;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.*;
+
+@Name("Node")
+@Description({
+	"Returns a node inside a config (or another section-node).",
+})
+@Examples({})
+@Since("INSERT VERSION")
+public class ExprNode extends PropertyExpression<Node, Node> {
+
+	static {
+		Skript.registerExpression(ExprNode.class, Node.class, ExpressionType.PROPERTY,
+				"[the] node %string% (of|in) %node%",
+				"%node%'[s] node %string%",
+				"[the] nodes (of|in) %nodes%",
+				"%node%'[s] nodes"
+		);
+	}
+
+	private boolean isPath;
+	private Expression<String> pathExpression;
```

Comment:
I would just call it `path`

Reply:
I typically like to leave the `expression` on the end because then I can use `path` as a variable when I actually get its value later without having to discriminate between the field and the variable :)